pack/trors_encounter.json

```diff
@@ -96,71 +65,45 @@
 		"quantity": 1,
 		"set_code": "hawkeye_nemesis",
 		"set_position": 3,
-		"spoiler": 1,
-		"text": "Attach to Crossfire. Otherwise, attach to the villain.<br>\n<b>Hero Action:</b> Exhaust your hero and spend a <span class=\"icon-wild\" title=\"Wild\"></span> resource → discard Crossfire's Rifle.",
-		"threat_fixed": false,
+		"text": "Attach to Crossfire. Otherwise, attach to the villain.\n<b>Hero Action:</b> Exhaust your hero and spend a <span class=\"icon-wild\" title=\"Wild\"></span> resource → discard Crossfire's Rifle.",
```

Comment:
```suggestion
		"text": "Attach to Crossfire. Otherwise, attach to the villain.\n<b>Hero Action:</b> Exhaust your hero and spend a [wild] resource → discard Crossfire's Rifle.",
```